db/rdbms/src/main/java/io/camunda/db/rdbms/write/service/SequenceFlowWriter.java

```diff
@@ -37,6 +37,16 @@ public void create(final SequenceFlowDbModel sequenceFlow) {
             sequenceFlow));
   }
 
+  public void insertIfNotExists(final SequenceFlowDbModel sequenceFlow) {
```

Comment:
💭 Name-wise, should we go for either `upsert` or `createIfNotExists` to be a but more consistent with the other writers? Not a big deal, really, just wondering.

Reply:
`createIfNotExists` does sound better.